aquadoggo/src/rpc/methods/entry_args.rs

```diff
@@ -95,15 +95,17 @@ mod tests {
     use crate::rpc::server::build_rpc_server;
     use crate::test_helpers::{
         handle_http, initialize_db, random_entry_hash, rpc_error, rpc_request, rpc_response,
+        TestClient,
     };
 
     const TEST_AUTHOR: &str = "8b52ae153142288402382fd6d9619e018978e015e6bc372b1b0c7bd40c6a240a";
 
-    #[async_std::test]
+    #[tokio::test]
     async fn respond_with_wrong_author_error() {
         let pool = initialize_db().await;
         let rpc_api = build_rpc_api_service(pool.clone());
```

Comment:
I see a lot of repition of this pattern make rpc api, then rpc server, then test client. Is that something that a test utility could shortcut? Something like `let client = TestClient::new()` that does all three steps?

Reply:
Yes, was wondering the same, but then I thought it might be too much optimization for something which gets removed anyhow .. I'd like to wait until we have the GraphQL API (and other fun things) and then improve the tooling based on future learnings.